pyzbar/pyzbar.py

```diff
@@ -144,9 +145,22 @@ def decode(image, symbols=None):
                 while symbol:
                     data = string_at(zbar_symbol_get_data(symbol))
                     symbol_type = ZBarSymbol(symbol.contents.value).name
+
+                    location = []
+                    loc = zbar_symbol_get_loc_size(symbol)
+                    if loc:
+                        location = [
+                            (
+                                zbar_symbol_get_loc_x(symbol, l),
+                                zbar_symbol_get_loc_y(symbol, l)
+                            )
+                            for l in range(loc)
```

Comment:
Prefer `xrange` on Python 2.x . At the top of the module (or maybe within this function):

```
RANGEFN = getattr(globals(), 'xrange', range)
```

Alter `range(loc)` to `RANGEFN(loc)`